ReactNativeClient/lib/models/Setting.js

```diff
@@ -1,3 +1,15 @@
+// *********************************************************************
+// **********************************************************************
+// ***                                                               ****
+// ***  Each build regenerates                                       ****
+// ***  ElectronClient/app/lib/models/Setting.js from                ****
+// ***  ReactNativeClient/lib/models/Setting.js, so only edit the    ****
+// ***  latter and not the former, otherwise you'll be confused      ****
+// ***  about why your code keeps disappearing.                      ****
+// ***                                                               ****
+// **********************************************************************
+// *********************************************************************
+
```

Comment:
Discussion: https://discourse.joplinapp.org/t/re-generated-electronclient-app-lib-models-setting-js-on-build/4429/1

Reply:
I missed that discussion earlier, I've added a comment over there.

Reply:
As mentioned in the discussion over there, adding the comment won't help since this issue applies to all the files, not just this one (and I prefer not to add a header to all the files). For now I would suggest we look into this issue separately from this pull request.

Reply:
Apologies, I did miss this! Thanks for [removing](https://github.com/laurent22/joplin/pull/2099#issuecomment-565251144).